### PR TITLE
Create cheatsheet

### LINKED ISSUE #10: Create cheatsheet for tuterm commands
Create a cheatsheet that can be used with the [`cheat`](https://github.com/cheat/cheat) program that:
- Shows the basic ways the `tuterm` command can be invoked
- Shows the most useful shell (API) functions that can be used within tuterm scripts (they are already documented in the [man page](https://github.com/veracioux/tuterm/blob/master/docs/tuterm.1)

### PR DESCRIPTION
**Pull request by smsalamon**
Resolves #10 

Cheatsheet with tuterm commands and shell functions that can be used with `cheat`.

### CODE REVIEW

**Comment by smsalamon** [comment]
@veracioux Let me know if I've missed anything or you want anything changed. Thanks!

File: docs/cheatsheet [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+evaluate <command>
+
+# In tutorial mode, to print a message to the user:
+message "your message"
+
+# To print a message to the user (tutorial and demo mode):
+Message "your message"
+
+# To check if the user passed an option to tuterm when starting a tutorial:
+option 'option'
```
Comment (veracioux) [inline docs/cheatsheet:40]: Change `'option'` to `<OPTION>` for consistency.

**Review by veracioux — CHANGES_REQUESTED** [review]
Thanks for your changes. I have left a review requesting some changes.

**Review by veracioux — APPROVED** [review]
@smsalamon Thanks. Merging now.

**Pull request merged**